server/src/test/java/io/spine/server/aggregate/EventApplierMethodTest.java

```diff
@@ -81,72 +91,99 @@ public void create_method() {
     }
 
     @Test
-    public void return_method_predicate() {
-        assertEquals(EventApplierMethod.predicate(), factory.getPredicate());
+    @DisplayName("allow invocation")
+    void invokeApplierMethod() throws InvocationTargetException {
+        final ValidApplier applierObject = new ValidApplier();
+        final EventApplierMethod applier = EventApplierMethod.from(applierObject.getMethod());
+        final RefProjectCreated event = Sample.messageOfType(RefProjectCreated.class);
+
+        applier.invoke(applierObject, event);
+
+        assertEquals(event, applierObject.eventApplied);
     }
 
     @Test
-    public void check_method_access_modifier() {
+    @DisplayName("check method access modifier")
+    void checkMethodAccessModifier() {
         final Method method = new ValidApplierButNotPackagePrivate().getMethod();
 
         factory.checkAccessModifier(method);
     }
 
-    @Test
-    public void consider_applier_with_one_msg_param_valid() {
-        final Method applier = new ValidApplier().getMethod();
+    @Nested
+    @DisplayName("consider applier valid")
+    class ConsiderApplierValid {
 
-        assertIsEventApplier(applier);
-    }
+        @Test
+        @DisplayName("when it has one message parameter")
+        void whenHasOneMessageParameter() {
+            final Method applier = new ValidApplier().getMethod();
 
-    @Test
-    public void consider_not_private_applier_valid() {
-        final Method method = new ValidApplierButNotPackagePrivate().getMethod();
+            assertIsEventApplier(applier);
+        }
 
-        assertIsEventApplier(method);
-    }
+        @Test
+        @DisplayName("when it's not private")
```

Comment:
I think `when` is a common part, that may be extracted to the parent display name.